pkg/odo/cli/service/create.go

```diff
@@ -281,3 +339,20 @@ func NewCmdServiceCreate(name, fullName string) *cobra.Command {
 	completion.RegisterCommandFlagHandler(serviceCreateCmd, "parameters", completion.ServiceParameterCompletionHandler)
 	return serviceCreateCmd
 }
+
+func gvFromALMExample(example map[string]interface{}) (group, version string) {
+	apiVersion := example["apiVersion"].(string)
+	gv := strings.Split(apiVersion, "/")
+
+	group, version = gv[0], gv[1]
+	return
+}
+
+func resourceFromCSV(csv olmv1alpha1.ClusterServiceVersion, crdName string) (resource string) {
+	for _, crd := range csv.Spec.CustomResourceDefinitions.Owned {
+		if crd.Kind == crdName {
+			resource = strings.Split(crd.Name, ".")[0]
```

Comment:
shouldn't the return happen instantaneous once we found the crd instead of looping over all? Are we considering the last one to match should be returned?

Reply:
This has been addressed although GH doesn't show it as "Outdated". Marking resolved.